server/modules/users/orders.js

```diff
@@ -521,19 +521,40 @@ function ProcessExchange(data)
 
                 exports.AddBalance(buyOrder.userID, buyerChange, sellOrder.price_pair, err => {
                     callback(err);
-                    ProcessComission(comission, sellOrder.price_pair);
+                    ProcessComission(comission, sellOrder.price_pair, buyOrder.userID, fromBuyerToSeller);
                 });
             });
         });
     }
     
-    function ProcessComission(comission, price_pair)
+    function ProcessComission(comission, price_pair, buerID, fromBuyerToSeller)
     {
-        for (var i=0; i<g_constants.DONATORS.length; i++)
-        {
-            if (g_constants.DONATORS[i].percent && g_constants.DONATORS[i].userID)
-                exports.AddBalance(g_constants.DONATORS[i].userID, (comission*(g_constants.DONATORS[i].percent*1-1)) / 100.0, price_pair, () => {});
-        }
+        const WHERE = 'userRegID="'+buerID+'"';
+        g_constants.dbTables['referals'].selectAll('*', WHERE, '', (err, rows) => {
+            const affiliateFee = (!err && rows && rows.length == 1) ? (comission / 2).toFixed(6)*1 : 0;
```

Comment:
Is this where the affiliate get 50% of commission?

Reply:
yes